dievai/functions/functions.core.php

```diff
@@ -333,4 +333,44 @@ function iconsMenu($icon)
 	$iconsMenu = applyFilters('adminMenuIcons', $iconsMenu);
 	
 	return ! empty($iconsMenu[$icon]) ? $iconsMenu[$icon] : null;
+}
+
+function getConfValue($key)
+{
+	global $prisijungimas_prie_mysql;
+
+	$request = "SELECT `val` FROM `" . LENTELES_PRIESAGA . "nustatymai` WHERE `key` = '$key'";
```

Comment:
```suggestion
	$request = "SELECT `val` FROM `" . LENTELES_PRIESAGA . "nustatymai` WHERE `key` = " . escape($key);
```
Reiketu nepasitiketi ivedamais duomenimis